pybamm/parameters/standard_parameters_lead_acid.py

```diff
@@ -91,9 +91,37 @@
 U_Pb_ref = pybamm.Parameter("U_Pb_ref")  # Reference OCP in the lead [V]
 U_PbO2_ref = pybamm.Parameter("U_PbO2_ref")  # Reference OCP in the lead dioxide [V]
 
+# --------------------------------------------------------------------------------------
+"""Scales"""
+Lx = pybamm.standard_parameters.Lx
+F = pybamm.standard_parameters.F
+R = pybamm.standard_parameters.R
+T = pybamm.standard_parameters.T
+
+concentration_scale = cmax
+length_scale = Lx
+time_scale = cmax * F * Lx / ibar  # discharge time scale
+potential_scale = R * T / F
+current_scale = ibar
+interfacial_current_scale_neg = ibar / (An * Lx)
+interfacial_current_scale_pos = ibar / (Ap * Lx)
+velocity_scale = ibar / (cmax * F)  # Reaction velocity scale
+
 # --------------------------------------------------------------------------------------
 """Functions"""
 
+
+def dimensional_current(t):
+    """Returns the dimensional current as a function of time """
+    t = pybamm.t
```

Comment:
Should this function take no arguments, or be a variable (since you are simply overwriting this argument anyway)?

Reply:
Oops, yeah, input will always be `pybamm.t` as far as I can think of so don't know whether it's clearer to make that explicit in the model (so this function takes a parameter) or leave it implicit